examples/e2ebot/data/stories.yml

```diff
@@ -27,12 +27,16 @@ stories:
   - action: utter_greet
   - intent: search_restaurant
   - action: utter_suggest_cuisine
-  - user: "I can always go for sushi"
+  - user: |
+      I can always go for sushi

```

Comment:
@Archish27 what's the reason for changing the format here?

Reply:
While training a bot receiving errors `intent` is not defined so had to make these changes.

Reply:
Hmm I just tried this and didn't get that error. Can you try changing it back and posting the error? I'd rather leave this in the former format since the e2e example is meant to highlight that kind of format

Reply:
reverted, ignore the issue was on my local setup